src/DependencyInjection/Configuration.php

```diff
@@ -91,26 +74,15 @@ public function getConfigTreeBuilder(): TreeBuilder
                 ->end()
 
                 ->scalarNode('context_manager')->defaultValue('sonata.block.context_manager.default')->end()
-                // NEXT_MAJOR: deprecate option and only allow setting it to false
-                ->arrayNode('http_cache')
-                    ->addDefaultsIfNotSet()
-                    ->beforeNormalization()
-                        ->always(function ($v) {
-                            if (false === $v) {
-                                $this->httpCacheDisabled = true;
-
-                                return [];
-                            }
-
-                            return $v;
-                        })
-                    ->end()
-                    ->children()
-                        // NEXT_MAJOR: remove option
-                        ->scalarNode('handler')->defaultValue('sonata.block.cache.handler.default')->end()
-                        // NEXT_MAJOR: remove option
-                        ->booleanNode('listener')->defaultTrue()->end()
-                    ->end()
+                // TODO: remove on 6.x
```

Comment:
NEXT_MAJOR

Reply:
I guess then someone might remove it on 5.x already and think its some 4.x leftover? :smile: